src/test/java/com/auth0/jwk/JwkTest.java

```diff
@@ -44,7 +46,16 @@ public void shouldBuildWithMap() throws Exception {
     @Test
     public void shouldReturnPublicKey() throws Exception {
         final String kid = randomKeyId();
-        Map<String, Object> values = publicKeyValues(kid);
+        Map<String, Object> values = publicKeyValues(kid, KEY_OPS_ARRAY);
+        Jwk jwk = Jwk.fromValues(values);
+
+        assertThat(jwk.getPublicKey(), notNullValue());
+    }
```

Comment:
add `assertThat(jwk.getOperations(), is(KEY_OPS_ARRAY))` or `listContaining..(KEY_OPS_ARRAY)`. Rename that constant to reflect it's a List rather than an array.